server/app/controllers/api/v1/wishes_controller.rb

```diff
@@ -46,11 +47,15 @@ def wish_params
   end
 
   def set_api_user
-    @api_user = ApiUser.find(params[:api_user_id])
+    if !params[:api_user_id].nil?
```

Comment:
Opportunity to use `#present?`

```suggestion
    if params[:api_user_id].present?
```

Reply:
Agree and updated